css/properties.json

```diff
@@ -6806,6 +6806,28 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/place-items"
   },
+  "place-self": {
+    "syntax": "<'align-self'> <'justify-self'>?",
+    "media": "visual",
+    "inherited": false,
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "CSS Box Alignment"
+    ],
+    "initial": [
+      "align-self",
+      "justify-self"
+    ],
+    "appliesto": "allElements",
```

Comment:
The spec says that it applies to "block-level boxes, absolutely-positioned boxes, and grid items", not all elements.

https://www.w3.org/TR/css-align-3/#place-self-property

Reply:
```suggestion
    "appliesto": "blockLevelBoxesAndAbsolutelyPositionedBoxesAndGridItems",
```